I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/tutorial_story.rpy", line 384, in script
    call screen map
  File "renpy/common/000statements.rpy", line 670, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
TypeError: Missing required positional arguments: 'place'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/tutorial_story.rpy", line 384, in script
    call screen map
  File "/Users/anastasiakozlova/renpy-8.1.3-sdk/renpy/ast.py", line 2259, in execute
    self.call("execute")
  File "/Users/anastasiakozlova/renpy-8.1.3-sdk/renpy/ast.py", line 2241, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "/Users/anastasiakozlova/renpy-8.1.3-sdk/renpy/statements.py", line 342, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 670, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "/Users/anastasiakozlova/renpy-8.1.3-sdk/renpy/exports.py", line 3347, in call_screen
    rv = renpy.ui.interact(mouse="screen", type="screen", roll_forward=roll_forward)
  File "/Users/anastasiakozlova/renpy-8.1.3-sdk/renpy/ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Users/anastasiakozlova/renpy-8.1.3-sdk/renpy/display/core.py", line 3582, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/Users/anastasiakozlova/renpy-8.1.3-sdk/renpy/display/core.py", line 4055, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Users/anastasiakozlova/renpy-8.1.3-sdk/renpy/display/core.py", line 684, in visit_all
    d.visit_all(callback, seen)
  File "/Users/anastasiakozlova/renpy-8.1.3-sdk/renpy/display/core.py", line 684, in visit_all
    d.visit_all(callback, seen)
  File "/Users/anastasiakozlova/renpy-8.1.3-sdk/renpy/display/core.py", line 684, in visit_all
    d.visit_all(callback, seen)
  File "/Users/anastasiakozlova/renpy-8.1.3-sdk/renpy/display/screen.py", line 476, in visit_all
    callback(self)
  File "/Users/anastasiakozlova/renpy-8.1.3-sdk/renpy/display/core.py", line 4055, in <lambda>
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Users/anastasiakozlova/renpy-8.1.3-sdk/renpy/display/screen.py", line 487, in per_interact
    self.update()
  File "/Users/anastasiakozlova/renpy-8.1.3-sdk/renpy/display/screen.py", line 680, in update
    self.screen.function(**self.scope)
  File "/Users/anastasiakozlova/renpy-8.1.3-sdk/renpy/ast.py", line 296, in apply_arguments
    return parameters.apply(args, kwargs, ignore_errors)
  File "/Users/anastasiakozlova/renpy-8.1.3-sdk/renpy/ast.py", line 232, in apply
    raise TypeError(
TypeError: Missing required positional arguments: 'place'

macOS-10.16-x86_64-i386-64bit x86_64
Ren'Py 8.1.3.23091805
Social Engineering Game 1.0
Thu Feb  8 11:25:23 2024
